src/data/markdown/docs/01 guides/03 Results visualization/05 InfluxDB - Grafana.md

```diff
@@ -124,6 +129,18 @@ $ docker-compose run -v \
 Now you should be able to connect to localhost on port 3000 with your browser and access the
 Grafana installation in the Docker container.
 
+## InfluxDB options
+
+When uploading the k6 results to InfluxDB (`k6 run --out influxdb=`), you can configure other InfluxDB options passing these environment variables:
+
+| InfluxDB Options             | Description                                               |  Default |
+| --------------------------------------- | ---------------------------------------------- | -----------------------| 
+| `K6_INFLUXDB_USERNAME`                  | Influxdb username, optional                     |    |
+| `K6_INFLUXDB_PASSWORD`                  | Influx user password                     |    |
+| `K6_INFLUXDB_INSECURE`                  | If `true`, it will skip https certificate verification  | `false`   |
+| `K6_INFLUXDB_TAGS_AS_FIELDS`            |  A comma-separated string to set k6 metrics as nonindexable fields (instead of tags). Example: `vu:int,iter:int,url:string,event_time:int`  |   |
```

Comment:
```suggestion
| `K6_INFLUXDB_TAGS_AS_FIELDS`            |  A comma-separated string to set k6 metrics as nonindexable [fields](https://docs.influxdata.com/influxdb/v1.8/concepts/glossary/#field) (instead of tags). An optional type can be specified using `:type` as in `vu:int` will make the field interger. The possible field types are `int`, `bool`, `float` and `string`, which is the default. Example: `vu:int,iter:int,url:string,event_time:int`  |   |
```